src/main/java/com/dev/cinema/model/dto/UserResponseDto.java

```diff
@@ -0,0 +1,9 @@
+package com.dev.cinema.model.dto;
+
+import lombok.Data;
+
+@Data
+public class UserResponseDto {
+    private Long id;
+    private String name;
```

Comment:
Maybe we'll need the user's email here as well)

Reply:
I'll let our user decide if he whether wants to hide email or not as long as it's required to login

Reply:
Actually there is nothing else i can add in this case 🤔